go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/mendersoftware/mender-snapshot
+
+go 1.20
```

Comment:
Q: Should this align with the `Yocto` version?

Reply:
This doesn't really matter for Yocto, it will build (or try to) with whatever version it is available in the distro.

Reply:
I just see that they reference it as:
```

minimum-go-version
    The minimum version of Go required to compile packages in this module. 
    ```
    
    which is not completely right then?
    
    Also, they say they will start enforcing this from `1.21` and onwards.
    
    [1] https://go.dev/doc/modules/gomod-ref

Reply:
Thank you. I launched now:
* https://github.com/mendersoftware/mender-setup/pull/4
* https://github.com/mendersoftware/mender-snapshot/pull/2

I set them to 1.17 based on the "required" version for `mender`.  I bet the hello world code can run on golang 1.0 :smile: